lib/datasource/docker/index.spec.ts

```diff
@@ -408,7 +407,8 @@ describe(getName(), () => {
           200,
           { tags },
           {
-            link: '<https://api.github.com/user/9287/repos?page=3&per_page=100>; rel="next", ',
+            link:
+              '<https://api.github.com/user/9287/repos?page=3&per_page=100>; rel="next", ',
```

Comment:
This url looks wrong to me either. Shouldn't it point to the docker registry?

Reply:
This was an incorrect lint change - this has not been adjusted in this PR.